docs/glossary.rst

```diff
@@ -447,6 +447,19 @@ In CrateDB Cloud, there are two distinct system :ref:`users <gloss-user>`:
     :ref:`Audit Log <overview-org-audit>`
 
 
+.. _gloss-tier:
+
+Tier
+----
+
+In the CrateDB Cloud :ref:`subscription plans <gloss-subscription-plan>`, tiers
+offer different magnitudes of the hardware composition of a given plan. For a
+given ratio of HD capacity, memory, and CPUs, going up in tier allows you to
+multiply the hardware values for your cluster deployment without changing the
+hardware ratio. Higher tiers do not increase the :ref:`DTU <gloss-DTU>` cost of
```

Comment:
I think this might be a bit misleading - the cost of the cluster does increase as you go to a higher tier, doesn't it? Perhaps omit it altogether?